documentation/docs/pages/inputs/restarts.md

```diff
@@ -0,0 +1,114 @@
+ESM1.6 restart files contain copies of complete model states, allowing for experiments to be stopped and restarted at a later time. This page outlines their structure and details basic procedures for manipulating them.
+
+
+## Structure of the ESM1.6 restart directory
+
+An ESM1.6 restart directory contains separate restart files for each component. These are are organised into the following directories:
+```
+atmosphere  coupler  ice  ocean  README
+```
+
+### Atmosphere
+The atmosphere restart directory contains the following files
+```
+restart_dump.astart  um.res.yaml
+```
+`restart_dump.astart` is the main UM restart file, containing the atmospheric state in addition to static boundary information such as the land-sea mask, vegetation maps and orography.
+
+`um.res.yaml` is a separate calendar file which holds the date and time associated with the restart. Information in this file is copied by `payu` into the model namelist files at runtime.
+
+
+### Ocean
+The ocean restart files are organised into seperate groups of variables:
+```
+ocean_age.res.nc  ocean_density.res.nc  ...  ocean_solo.res
+```
+
+The netCDF files contain snapshots of the model state while `ocean_solo.res` is a text file specifying the model date and time.
+
+### Ice
+
+The ice restart directory contains the following files:
+
+```
+cice_in.nml  iced.2145-01-01-00000.nc  ice.restart_file  mice.nc
+```
+
+The `iced.YYYY-MM-DD-00000.nc` holds the model state and also carries the date and time in the global attributes. `ice.restart_file` is 
+a pointer file used by CICE when finding the correct restart file to read. `mice.nc` contains ice coupling data to be sent to the ocean model
+at the beginning of the next run.
+
+### Coupler
+The coupler restart directory contains data sent from each component at the termination of the previous run, allowing for the submodels to access the required boundary conditions at the beginning of the next run.
+
+```
+a2i.nc  i2a.nc  o2i.nc
+```
+
+
+
+## Common restart manipulations
+
+### Changing the date of a restart file
+
+It's commonly required to change the date for a restart file. For example when setting up a historical experiment, a restart might be taken from a pre-industrial simulation and the date changed to 1850.
+
+The following instructions outline how to modify the date for each component. It's important to apply the updates to all components consistently, as inconsistencies in the model dates can cause crashes.
+
+#### Atmosphere:
```

Comment:
Beforehand, should there be a step to copy the chosen restart folders for each model to a new location? We don't want people to change restarts in place. 

Maybe:

```markdown
!!! info

    Before starting any manipulation, copy the restart folders you want to modify to a new location of your choice.
```